### PR TITLE
fix(OverflowMenu): fix border-radius and add larger size options

### PR DESCRIPTION
**Pull request by tjmabey**
This addresses https://github.com/InsideSalesOfficial/insidesales-components/issues/225

**Changes:**
* fix corners not being rounded
* adjust border-radius to 3px by request from Kellie
* add medium and large props for larger menu options
* add key to SubmenuOptionsWrapper component

**before:**
Screen Shot 2019-03-27 at 5 23 42 PM

**after:**
default
Screen Shot 2019-03-27 at 5 10 48 PM
medium
Screen Shot 2019-03-27 at 5 11 17 PM
large
Screen Shot 2019-03-27 at 5 11 44 PM

### CODE REVIEW

**Comment by aar0nwils0n** [comment]
They currently resize dynamically based on the content. Is there an issue with this approach or special use case for having fixed sizing?

**Comment by tjmabey** [comment]
I recently chatted with Kellie Shelton about this. She gave us an updated design that has an overflowmenu at 160px width. Based on past commit history (https://github.com/InsideSalesOfficial/insidesales-components/commit/bc73d61f80848da105f453946d59a6c9fc39474d), I thought it would be better to just make several different sizes available in case design changes their preference again

File: src/components/OverflowMenu/OverflowMenu.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const SelectOption = styled.div`
 
   box-sizing: border-box;
   display: flex;
-  padding: 0 10px;
+  padding: 0 24px 0 10px;
```
Comment (aar0nwils0n) [inline src/components/OverflowMenu/OverflowMenu.js]: I'm curious why the right side needs different padding. If this is the case we would probably need to adjust for which direction the menu is facing

**Comment by tjmabey** [comment]
Updated PR:
- removed larger menu size options
- the min-width will continue to be 120px
- added a little padding to right edge of menu options

this is from Kellie:
> minimum width will be 120 and then everything else will dynamically resize. So there will be 24px of padding from the right edge of the longest word in an overflow menu

File: src/components/OverflowMenu/OverflowMenu.js (hunk 1 above)
Comment (tjmabey) [reply src/components/OverflowMenu/OverflowMenu.js]: you beat me to commenting... kellie asked for it but i see what you mean now about direction..

Comment (tjmabey) [reply src/components/OverflowMenu/OverflowMenu.js]: i dont actually think the direction matters for this. kellie asked for 24px padding on the right edge. what would you like me to do?

Comment (aar0nwils0n) [reply src/components/OverflowMenu/OverflowMenu.js]: Ah, I suppose text text is always right aligned anyway so it doesn't matter. It's fine as is

**Comment by tjmabey** [comment]
requesting additional review @t0v1n @richardbolt @austinknight @nathguen @gabe-lei @cozzbp @thundercat1 @jeffhoffman1 @rsmorley

**Pull request merged**